packages/core/test/after.test.ts

```diff
@@ -118,6 +118,36 @@ describe('delayed transitions', () => {
       .start();
   });
 
+  it('should defer a single send event for a delayed transition with multiple conditions (#886)', () => {
```

Comment:
this is not quite a good description - there are no multiple conditions defined here, but I'm having trouble with rephrasing this

Reply:
I'm not sure either, this is fine for now.